tests/e2e/scenarios/scalability/run-test.sh

```diff
@@ -17,6 +17,35 @@
 set -e
 set -x
 
+# special processing for ASG force a delete and wait for it to go away
+delete_asg_with_timeout() {
+    local ASG_NAME=$1
+    local TIMEOUT=900  # 15 minutes in seconds
```

Comment:
For 5k nodes, this usually takes 1h or so. This is why in kOps, after we delete the ASG, we start terminating instances in batches.